src/components/series.tsx

```diff
@@ -42,10 +42,10 @@ const immersiveStyles = ({ kicker }: PillarStyles, isLabs: boolean): SerializedS
 const font = (isLabs: boolean): SerializedStyles => {
     return css`
         ${
-            isLabs
-                ? textSans.medium({ lineHeight: 'loose', fontWeight: 'bold' })
-                : headline.xxxsmall({ lineHeight: 'loose', fontWeight: 'bold' })
-        }`
+    isLabs
+        ? textSans.medium({ lineHeight: 'loose', fontWeight: 'bold' })
+        : headline.xxxsmall({ lineHeight: 'loose', fontWeight: 'bold' })
+}`
```

Comment:
Is this alignment correct?

Reply:
Yeah... ESLint sometimes seems to get a bit confused by template strings... I've refactored the two areas you flagged, let me know what you think!